cmd/acra-translator/grpc_api/service_test.go

```diff
@@ -669,35 +651,6 @@ func testgRPCServiceFlow(ctx context.Context, expectedClientID []byte, conn *grp
 	_, _ = searchableClient.GenerateQueryHash(ctx, &QueryHashRequest{Data: testData})
 }
 
-func TestNewFactoryWithClientIDFromSecureSessionConnectionSuccess(t *testing.T) {
-	expectedClientID := []byte("some id")
-	testKeypair, err := keys.New(keys.TypeEC)
-	if err != nil {
-		t.Fatal(err)
-	}
-	keystorage := &mocks.TranslationKeyStore{}
-
-	ctx, cancel := context.WithTimeout(context.Background(), connectionTimeout*5)
-	defer cancel()
-
-	data := &translatorCommon.TranslatorData{UseConnectionClientID: true, Keystorage: keystorage, Tokenizer: newTokenizer(t)}
-	secureSessionWrapper, err := network.NewSecureSessionConnectionWrapper(expectedClientID, keystorage)
-	if err != nil {
-		t.Fatal(err)
-	}
-	secureSessionWrapper.AddOnServerHandshakeCallback(network.TraceConnectionCallback{})
-	server := newServer(data, secureSessionWrapper, t)
-	defer server.Stop()
-	clientOpts := []grpc.DialOption{getSecureSessionDialer(secureSessionWrapper), grpc.WithInsecure()}
-
-	keystorage.On("GetPrivateKey", mock.Anything).Return(testKeypair.Private, nil)
-	keystorage.On("GetPeerPublicKey", mock.Anything).Return(testKeypair.Public, nil)
-	conn := server.NewConnection(clientOpts, t)
-	defer conn.Close()
-
-	testgRPCServiceFlow(ctx, expectedClientID, conn, keystorage, t)
-}
-
 func TestNewFactoryWithClientIDFromSecureSessionConnectionInvalidAuthInfo(t *testing.T) {
```

Comment:
this one we can remove too. but only if we have `<Something else>InvalidAuthInfo` check, to be sure that we handle case with incorrect auth info. But if not, then better to replace SS wrapper with TLS/something else and remove SS usage too